File: src/assets/usage-examples-code/windows/has_window/has_window-1-basic-usage.cpp
#include "splashkit.h"

int main()
{
    string caption = "Test Window";
    open_window(caption, 800, 600);

    if (has_window(caption))
    {
        write_line("A window with the caption '" + caption + "' exists.");
    }
    else
    {
        write_line("No window with the caption '" + caption + "' exists.");
    }

    return 0;
}

File: src/assets/usage-examples-code/windows/has_window/has_window-1-basic-usage.cs
using System;
using SplashKitSDK;

string caption = "Test Window";
SplashKit.OpenWindow(caption, 800, 600);

if (SplashKit.HasWindow(caption))
{
    Console.WriteLine("A window with the caption '" + caption + "' exists.");
}
else
{
    Console.WriteLine("No window with the caption '" + caption + "' exists.");
}
